docs/package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "@docusaurus/core": "2.4.0",
     "@docusaurus/preset-classic": "2.4.0",
-    "@easyops-cn/docusaurus-search-local": "^0.35.0",
+    "@easyops-cn/docusaurus-search-local": "^0.29.0",
```

Comment:
Why did we downgrade?

Reply:
This is not required anymore! Earlier when I had a broken link, the downgrade somehow made it possible for docusaurus to still be able to find the sphinx docs.